### PR TITLE
Add loading and error handling

### PR DESCRIPTION
**Pull request by Caruki**
Added loading and error state with corresponding visual alerts for the user

### CODE REVIEW

File: src/pages/Result.js [hunk 1]
```diff
@@ -55,6 +82,8 @@ export default function Result() {
         name="Go back to overview"
         destination="/"
       ></RedirectButton>
+  </>
```
Comment (lmachens) [inline src/pages/Result.js]: Looks like prettier is not working here. Maybe there is an error in this file?

File: src/components/LoadingAnimation.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  height: 60px;
+  top: 20px;
+  left: 20px;
+  border-radius: 50%;
+  box-shadow: 0 2px 0 0 #93dbe9;
+  transform-origin: 30px 31px;
+  box-sizing: content-box;
+`;
+
+const LoadingSpinH = styled.div`
```
Comment (lmachens) [inline src/components/LoadingAnimation.js]: What does the `H` stand for? Maybe find a better name.

**Review by lmachens — COMMENTED** [review]
gj

File: src/pages/Result.js (hunk 1 above)
Comment (Caruki) [reply src/pages/Result.js]: I think I just forgave to save... 😅

**Pull request merged**